qiskit_optimization/translators/ising.py

```diff
@@ -131,6 +150,12 @@ def from_ising(
     i-th variable is mapped to i-th qubit.
     See https://github.com/Qiskit/qiskit-terra/issues/1148 for details.
 
+    .. note::
+
+        The ``qubit_op`` argument can currently accept Opflow operators, but it has been
+        superseded by ``SparsePauliOp``.
+        Opflow will be deprecated in a future release and subsequently removed after that.
```

Comment:
```suggestion
        The ``qubit_op`` argument can currently accept Opflow operators (OperatorBase type), but have been
        superseded by Qiskit Terra quantum info BaseOperators such as  ``SparsePauliOp``.
        Opflow operator support will be deprecated in a future release and subsequently removed after that.
```
Here is accepts all quantum info BaseOperator types right. Most often it will be SparsePauliOp bit its not restricted to this. I will also note that we can make these class references since we have the intersphinx with Terra.

Reply:
Thanks. I include it. cc41551